lib/experimental/webfonts.php

```diff
@@ -211,6 +240,47 @@ function wp_get_webfont_providers() {
 	}
 }
 
+if ( ! function_exists( 'wp_print_webfonts' ) ) {
+	/**
+	 * Invokes each provider to process and print its styles.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param string|string[]|false $handles Optional. Items to be processed: queue (false),
+	 *                                       single item (string), or multiple items (array of strings).
+	 *                                       Default false.
+	 * @return array|string[] Array of web font handles that have been processed.
+	 *                        An empty array if none were processed.
+	 */
+	function wp_print_webfonts( $handles = false ) {
+		global $wp_webfonts;
+
+		/**
+		 * Fires before webfonts in the $handles queue are printed.
+		 *
+		 * @since 6.1.0
+		 */
+		do_action( 'wp_print_webfonts' );
```

Comment:
Why do we need this action?
This function gets hooked in `wp_head` and `admin_print_styles`, so can't plugin/theme authors use those hooks already to do stuff, and utilize the priorities to run their hooks before/after the webfonts?

Reply:
I added this for parity with scripts and styles.

You're not wrong, but in my opinion, it's much easier to have a specific hook to use than having to figure out the priorities of others.

Reply:
I agree. I saw the parity but as @aristath notes it's not needed. Will adjust. Thanks both you!

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/42946/commits/c0c0c3db862c8805e04a4a68e63edd3f1237bde7.